### ISSUE TITLE
issue with StreamingTokenDecoder?

### ISSUE THREAD

**Issue by MagicMaxxx** [SciSharp/LLamaSharp#483]
I asked this: "что ты знаешь про 105 статью УК РФ?" (also it was several other questions in Russian)
then I got an exception coming from StreamingTokenDecoder - "System.ArgumentOutOfRangeException: 'Specified argument was out of the range of valid values.'":

Model: openbuddy-llama2-13b64k-v15.Q4_K_M.gguf

![issue with StreamingTokenDecoder](https://github.com/SciSharp/LLamaSharp/assets/19684558/8999232e-4c3f-4c8b-bbe2-4d512fbf7d29)

**Comment by MagicMaxxx** [comment]
ok, seems like it was already fixed in the repository, but not published in the Nuget package yet. Waiting for the fix!

Thanks!

**Comment by MagicMaxxx** [comment]
Hello!
I have a question - when do you plan to release the next version? Problem that when I'm using the current Nuget-packages (0.9.1), due to problem in the StreamingTokenDecoder I have my code crashed on the 2nd-3rd prompt... 
Thanks!

**Comment by martindevans** [comment]
It's held up on some backend changes at the moment. If those aren't ready within a few days we'll probably release without them.

**Comment by MagicMaxxx** [comment]
thanks for the info!

BTW - I tried a different approach - I added LlamaSharp+LlamaSharp.Sem.Kernel C# projects right into my sln (via "Add existing project") but it didn't start Debug as it seems like Llama.Backend.* project (added via Nuget) requires exactly 0.9.1 version... Not sure about it but this approach didn't work for me. So I still need to wait until the new release.

Again - thanks for the info!
